DashKit/Pod.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Pod.swift
+//  DashKit
+//
+//  Created by Pete Schwamb on 4/18/19.
+//  Copyright © 2019 Tidepool. All rights reserved.
+//
+
+import Foundation
+
+public struct Pod {
+    // Volume of insulin in one motor pulse
+    public static let pulseSize: Double = 0.05
+
+    // Number of pulses required to delivery one unit of insulin
+    public static let pulsesPerUnit: Double = 20
```

Comment:
Shouldn't this be depended on the `pulseSize`?

Reply:
Yeah, I will express it that way (1/pulseSize).